Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -109,7 +109,28 @@ class SentryUserFeedbackForm: UIViewController {
     }
     
     func submitFeedbackButtonTapped() {
-        // TODO: validate and package entries
+        var missing = [String]()
+        
+        if config.formConfig.isNameRequired && fullNameTextField.hasText {
+            missing.append("name")
+        }
+        
+        if config.formConfig.isEmailRequired && emailTextField.hasText {
+            missing.append("email")
+        }
+        
+        if !messageTextView.hasText {
+            missing.append("description")
+        }
+        
+        guard missing.isEmpty else {
+            let list = missing.count == 1 ? missing[0] : missing[0 ..< missing.count - 1].joined(separator: ", ") + " and " + missing[missing.count - 1]
+            let alert = UIAlertController(title: "Error", message: "You must provide all required information. Please check the following fields: \(list).", preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "OK", style: .default))
+            present(alert, animated: config.widgetConfig.animations)
```

Comment:
`l`: I'm curious. Do we need localization for this at some point?

Reply:
I just mentioned this the other day but it was never officially discussed yet, and I don't know what is currently being done on the Javascript side. Presumably we would want it one day. wdyt @bruno-garcia ?

Reply:
Step by step. We do it once people ask for it.